packages/rpcview.vm/rpcview.vm.nuspec

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>rpcview.vm</id>
+    <version>0.3.1.20231018</version>
+    <authors>silverf0x</authors>
+    <description>RpcView is an open-source tool to explore and decompile all RPC functionalities present on a Microsoft system</description>
+    <dependencies>
+      <dependency id="common.vm" />
+    </dependencies>
```

Comment:
This package requires "Microsoft Visual C++ 2019 Redistributable" in order to work. Adding this line under `<dependency id="common.vm" />` will make it work:

`<dependency id="vcredist140" version="14.36.32532"/>`